### DISCUSSION TITLE
Prune Doesn't Appear to be Reclaiming Disk Space

### DISCUSSION THREAD

**Discussion by ghost** [erigontech/erigon#6791] [Q&A]
I've been running my execution client on the Mainnet with the "--prune htc" flag, but I am not sure how much pruning is actually taking place.  The logs say I currently have 347 GB in "ReclaimableSpace" and that number keeps slowly increasing.  The total amount of disk space now used by my client is up to 1.6 TB

**Comment by AskAlexSharov** [comment]
Do you do any analytics/slow requests? Do you read db directly?

**Comment by ghost** [comment]
I do not run any analytics. The execution client and the consensus client are both running on the same system.  The system only has 16GB RAM, could this be causing the issue?

**Reply by AskAlexSharov** [reply]
I think not related to ram
